pom.xml

```diff
@@ -15,6 +15,7 @@
 
   <properties>
     <dependency.pentaho-kettle.version>${project.version}</dependency.pentaho-kettle.version>
+    <dependency.pentaho-xul.revision>TRUNK-SNAPSHOT</dependency.pentaho-xul.revision>
```

Comment:
@pentaho-nbaker can you confirm this should be 6.0-SNAPSHOT?  I had left the version formerly present and only changed the prop name.